tests/testthat/setup.R

```diff
@@ -2,6 +2,9 @@
 library(glmnet)
 glmnet.control(fdev = 0)
 
+# use ncvreg for penalty comparison
+ncvreg <- ncvreg::ncvreg
```

Comment:
Why do this instead of loading `ncvreg` the same way that `glmnet` is loaded?

Reply:
Hi mentor, ncvreg's namespace also give definition for standardize, which causes conflicts with ours. My console raise warnings when I import ncvreg after sgdnet. It seems fine if I used ncvreg in this way.

Reply:
We have a `standardize` function? 

My search doesn't pick it up (https://github.com/jolars/sgdnet/search?l=R&q=standardize&type=)

Reply:
Hi mentor! Because loading `ncvreg` will mask datasets `heart` and `prostate` in our package. These two datasets  in `ncvreg` have different formats and will make former test cases using `heart` and `prostate` fail.